docs/core/tutorials/consuming-library-with-visual-studio.md

```diff
@@ -1,104 +1,83 @@
 ---
-title: Consume a .NET Standard library in Visual Studio 2017
-description: Build a .NET Core application that calls members of another class library with Visual Studio 2017.
+title: Consume a .NET Standard library in Visual Studio
+description: Build a .NET Core application that calls members of another class library with Visual Studio 2019.
 author: BillWagner
 ms.author: wiwagn
 ms.date: 06/05/2018
+dev_langs:
+  - "csharp"
+  - "vb"
 ms.custom: "vs-dotnet, seodec18"
 ---
-# Consume a .NET Standard library in Visual Studio 2017
+# Consume a .NET Standard library in Visual Studio
 
-Once you've created a .NET Standard class library by following the steps in [Create a class library with .NET Standard in Visual Studio](./library-with-visual-studio.md), tested it in [Test a class library with .NET Core in Visual Studio](testing-library-with-visual-studio.md), and built a Release version of the library, the next step is to make it available to callers. You can do this in two ways:
+Once you've created a .NET Standard class library, tested it, and built a release version of the library, the next step is to make it available to callers. You can do this in two ways:
 
-* If the library will be used by a single solution (for example, if it's a component in a single large application), you can include it as a project in your solution.
+- If the library will be used by a single solution (for example, if it's a component in a single large application), you can include it as a project in your solution.
+- If the library will be publicly available, you can distribute it as a NuGet package.
 
-* If the library will be generally accessible, you can distribute it as a NuGet package.
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+>
+> - Add a console app to your solution that references a .NET Standard library project.
+> - Create a NuGet package that contains a .NET Standard library project.
```

Comment:
Given that the tutorial title says it's about consuming a library,  it seems like the "create a package" section should have instructions on how to consume the package as well.

Reply:
Good point @tdykstra. I might need to think of a better title. Some of the headings were also misleading. I think adding instructions could be an enhancement for later on beyond the scope of updating these topics for VS 2019.

Reply:
Opened a new issue to track this: https://github.com/dotnet/docs/issues/16369